src/launcher/util/checkAppCompatibility.jsx

```diff
@@ -42,108 +42,137 @@ import requiredVersionOfShared from '../../main/requiredVersionOfShared';
 
 const config = remote.require('../main/config');
 
-const requiresMoreRecentVersionOfShared = (
-    requestedVersionOfShared,
-    providedVersionOfShared
-) =>
-    requestedVersionOfShared != null &&
-    semver.gt(requestedVersionOfShared, providedVersionOfShared);
+const isVersionNumber = versionNumberString =>
```

Comment:
Just a tiny comment. `isVersionNumber` is a bit confusing at my first glance. Maybe isVersionNumberValid/isValidVersion/isValidVersionNumber?

Reply:
Good idea. Did that in 2e2e1e0, a commit in #520.